test/Table_Tests/src/Common_Table_Operations/Select_Columns_Spec.enso

```diff
@@ -459,7 +459,7 @@ spec setup =
         Test.specify "should correctly handle problems: duplicate names" <|
             map = ["Test", "Test", "Test", "Test"]
             action = table.rename_columns map on_problems=_
-            tester = expect_column_names ["Test", "Test 1", "Test 2", "Test 3"]
+            tester = expect_column_names ["Test 1", "Test 2", "Test 3", "Test"]
             problems = [Duplicate_Output_Column_Names.Error ["Test", "Test", "Test"]]
             Problems.test_problem_handling action problems tester
```

Comment:
I think I'm slightly confused.

Changing the ordering of elements of the Map shouldn't really change the way `rename_columns` works, at least in theory. I guess that if it did, we should look into it.

I believe the ordering is not most crucial, but the original behaviour presented in this test seems rather preferred - we want the column without suffix to be the first column on the list, not last.

@jdunkerley shall I create a small bug report so that we can amend `rename_columns` to work again like before with the new Map?